### DISCUSSION TITLE
Not able to decrypt PGP Binary File with openpgpjs

### DISCUSSION THREAD

**Discussion by nnshahIVP** [openpgpjs/openpgpjs#1747] [Q&A] [answered]
I have an OpenPGP binary file. I want to decrypt the file with OpenPGP module and save the decrypted file in a folder. But when I am trying to run the code it does create the file but it can't be opened.

To see more information on the question, please see the below link,

https://stackoverflow.com/questions/78349261/not-able-to-decrypt-pgp-binary-file-with-openpgp-module

Thanks.

**Comment by larabr** [comment, marked as answer]
Hi, you are not writing a binary file, not sure if that works for ZIPs.
Try passing `format: binary` to the `openpgp.decrypt` options, otherwise UTF-8 (string) data is returned, and set the encoding option of `writeFileSync` to `null`.

**Reply by nnshahIVP** [reply]
It worked. Thank you.
